emily/handler/src/database/entries/deposit.rs

```diff
@@ -647,7 +647,10 @@ pub struct ValidatedUpdateDepositsRequest {
     ///
     /// This allows the updates to be executed in chronological order but returned in the order
     /// that the client sent them.
-    pub deposits: Vec<(usize, ValidatedDepositUpdate)>,
+    pub deposits: Vec<(
+        usize,
+        Result<ValidatedDepositUpdate, crate::common::error::ValidationError>,
```

Comment:
Usually we just import the type instead of using the full path. Any particular reason for doing it this way?

Reply:
[f36db4f](https://github.com/stacks-sbtc/sbtc/pull/1653/commits/f36db4fdbee7619b946abfa6a11be7ec4ee05c51)